spec/tbd_tests_spec.rb

```diff
@@ -2039,9 +2040,11 @@
     expect(model).to_not be_empty
     model = model.get
 
+    # Ensure the plenum is 'unoccupied', i.e. not part of the total floor area.
     plnum = model.getSpaceByName("PLENUM-1")
     expect(plnum).to_not be_empty
     plnum = plnum.get
+    expect(plnum.setPartofTotalFloorArea(false)).to be true
```

Comment:
Key in having TBD correctly identify _in/directly-conditioned_ spaces as 'unoccupied'.